tools/api-markdown-documenter/src/test/HtmlEndToEnd.test.ts

```diff
@@ -43,7 +44,7 @@ const snapshotsDirectoryPath = Path.resolve(
 
 // Relative to lib/test
 const testDataDirectoryPath = Path.resolve(dirname, "..", "..", "src", "test", "test-data");
-const testModelPaths = [Path.resolve(testDataDirectoryPath, "simple-suite-test.json")];
```

Comment:
Note: there is a fair amount of infra duplication between this file and `MarkdownEndToEndTests`. I plan on doing some cleanup here in a subsequent PR.